app/angular/src/server/framework-preset-angular-cli.ts

```diff
@@ -17,8 +17,8 @@ export async function webpackFinal(baseConfig: webpack.Configuration, options: P
     return baseConfig;
   }
 
-  const packageJson = await import(findUpSync('package.json', { cwd: options.configDir }));
-  const angularCliVersion = semver.coerce(packageJson.devDependencies['@angular/cli'])?.version;
+  // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Maybe we should add  `@angular/cli` in `peerDependencies` with optional `peerDependenciesMeta` and also add it in `devDependencies` 🤔

Reply:
yeah, fair point. Let me add that